src/Microsoft.Health.Dicom.Blob/Features/Health/DicomConnectedStoreHealthCheck.cs

```diff
@@ -84,21 +80,21 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
         }
         finally
         {
-            // Just in case the blob was successfully created but failed to be deleted, try clean up again
-            await TryDeleteBlob(blockBlobClient, cancellationToken);
+            await TryDeleteAsync(healthCheckBlobClient, cancellationToken);
```

Comment:
I think we can ignore this now, right? Do we need this `finally` block?

Reply:
I had left it in to have the health check do a best-effort clean up of the blob, and the expiry header would be our last line of defense to delete it an hour later.

But you're right, since the expiry header will always work, maybe I can get rid of this and just reduce the expiry time so there aren't too many blobs waiting to be deleted at once.